src/main/java/com/iexec/common/sdk/marketplace/Marketplace.java

```diff
@@ -31,6 +31,12 @@ public interface Marketplace {
 
     public List<WorkerpoolMarketOrder> getWorkerpoolOrders(String workerpoolAddress);
 
+    List<WorkerpoolMarketOrder> getTaggedWorkerpoolOrders(String tag);
```

Comment:
```suggestion
    List<WorkerpoolMarketOrder> getWorkerpoolOrders(String tag);
```
or 
```suggestion
    List<WorkerpoolMarketOrder> getWorkerpoolOrdersByTag(String tag);
```

Reply:
I prefer the second suggestion.

Reply:
Thanks, second suggestion implemented in https://github.com/iExecBlockchainComputing/iexec-common/pull/318/commits/11bc56acf27c534250addf237cbc2ba669c2e989.

Reply:
Suggestion #2 is not consistent with 
`public List<WorkerpoolMarketOrder> getWorkerpoolOrders(String workerpoolAddress);`
We shall upgrade one or the other

Reply:
You're right. Let's add an alias to `getWorkerpoolOrders` and deprecate this current method.